src/main/java/com/project/bookstore/validation/Email.java

```diff
@@ -0,0 +1,18 @@
+package com.project.bookstore.validation;
+
+import jakarta.validation.Constraint;
+import jakarta.validation.Payload;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Constraint(validatedBy = EmailValidator.class)
+@Target({ElementType.PARAMETER, ElementType.FIELD})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Email {

```

Comment:
jakarta.validation has own implementation of @Email. 

1. Use existing once solution to reduce amount of code. 
2. use existing once solution because they already well tested.